lib/rust/ensogl/component/grid-view/src/column_widths.rs

```diff
@@ -0,0 +1,91 @@
+//! Storage for column's widths in the GridView. See [`ColumnWidths`].
+
+use crate::prelude::*;
+use std::fmt::Formatter;
+
+
+
+// ====================
+// === ColumnWidths ===
+// ====================
+
+
+// === Helper definitions ===
+
+type SegmentTree = segment_tree::PrefixPoint<f32, segment_tree::ops::Add>;
+
+fn segment_tree(width_diffs: Vec<f32>) -> SegmentTree {
+    segment_tree::PrefixPoint::build(width_diffs, segment_tree::ops::Add)
+}
+
+
+// === ColumnWidths ===
+
+/// Storage for column widths.
+///
+/// It stores not the actual widths but rather the differences between [`Frp::set_entry_size`] and
+/// the width updated through [`Frp::set_column_width`] or [`EntryFrp::override_column_width`].
```

Comment:
`set_entry_size` is a size, so it should rather go "between the default width set by [`Frp::set_entry_size`] endpoint"